supabase/functions/users/index.ts

```diff
@@ -103,6 +124,10 @@ async function patchUser(req: Request, currentUserSale: any) {
         return createErrorResponse(500, 'Internal Server Error');
     }
 
+    if (avatar) {
+        await updateSaleAvatar(data.user.id, avatar);
```

Comment:
Something is wrong. This edge function is still accessible to anonymous users using the [remote function API](https://supabase.com/docs/guides/functions/deploy#invoking-remote-functions) right?

Reply:
It does, see:
- https://github.com/marmelab/atomic-crm/pull/74/files/b82310e675e8bb18b51169e671f5c45b78426dd0..2c2ed26627070fa5c34390eb25f10beca50467a1#diff-1d0b413d00a6df280ee14f49c3f5766d5ac5bd57ad21fdb338334bc15c3093baR111
- https://github.com/marmelab/atomic-crm/pull/74/files#diff-1d0b413d00a6df280ee14f49c3f5766d5ac5bd57ad21fdb338334bc15c3093baR189-R197

Reply:
My bad, you're right!